src/Factory/ProgrammeResponseFactory.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Factory;
+
+use App\Controller\Helper\ProgrammeRequestContentType;
+use App\Repository\ProgrammeRepository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Serializer\SerializerInterface;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProgrammeResponseFactory
```

Comment:
I don't see the benefit of having this class other than moving code. This class has two methods that take a request and build a response (so basically they are controllers without being declared as controllers).